packages/office-ui-fabric-react/src/components/FocusTrapZone/FocusTrapZone.types.ts

```diff
@@ -54,6 +67,14 @@ export interface IFocusTrapZoneProps extends React.HTMLAttributes<HTMLDivElement
    */
   disableFirstFocus?: boolean;
 
+  /**
+   * If the FocusTrapZone is declared focusable (data-is-focusable=true) and it
+   * receives focus directly, or if .focus() is called, it will pass on
+   * focus to a child element.  This prop determines how that element is selected.
+   * @default FocusTransferRule.firstFocusable
+   */
+  focusTransferRule?: FocusTransferRule;
+
   /**
    * Optional, onKeyDown event handler
```

Comment:
From FocusTrapZone it looks like you added a onFocusCapture prop, but I'm not seeing it here

Reply:
It's part of divProps.  Event handlers should always allow chaining.

Reply:
In that case why don't we just use nativeProps and get all the divProps that the creator wants to pass in

Reply:
I agree.  Isn't that what's happening?  Help me understand what I'm missing.

It's not clear to me why onKeyDown was included in FTZ.types.ts since all divProps are implicitly props on the control and the docs mention that with allowNativeProps=true.

Reply:
Yeah, I don't think onKeyDown is needed anymore now that the divProps are used